tools/setup-envtest/workflows/workflows.go

```diff
@@ -85,3 +87,23 @@ func (f Sideload) Do(env *envp.Env) {
 	env.Sideload(ctx, f.Input)
 	env.PrintInfo(f.PrintFormat)
 }
+
+// Version is the workflow that shows the current binary version
+// of setup-envtest.
+type Version struct {
+	BinaryVersion string
+}
+
+// Do executes the workflow.
+func (v Version) Do(env *envp.Env) {
+	version := "(unknown)"
```

Comment:
Maybe also move this logic to the version pkg I suggested in another comment. Then the stamping and this logic is in the same place

Maybe we can just use literally the same version.go as in controller-gen (+/- controller-tools specific godoc)? and then call version.Version() here instead of passing the version through?

(this func here would then ~ only contain the print statement)